rq/job.py

```diff
@@ -360,12 +361,12 @@ def set_id(self, value):
     @classmethod
     def key_for(cls, job_id):
         """The Redis key that is used to store job hash under."""
-        return b'rq:job:' + job_id.encode('utf-8')
+        return (cls.redis_job_namespace_prefix + job_id).encode('utf-8')
 
     @classmethod
     def dependents_key_for(cls, job_id):
         """The Redis key that is used to store job dependents hash under."""
-        return 'rq:job:{0}:dependents'.format(job_id)
+        return '{0}{0}:dependents'.format(cls.redis_job_namespace_prefix, job_id)
```

Comment:
Could you please fix this typo? Should be `{0}{1}`

Reply:
Will do. Saw that I did same mistake for `COMPACT_QUEUE`.

Do you know why `key_for` returns a binary while `dependents_key_for` returns string?